frontend/libs/studio-components-legacy/src/components/StudioCodelistEditor/utils.test.ts

```diff
@@ -144,4 +146,21 @@ describe('StudioCodelistEditor utils', () => {
       expect(isCodeLimitReached(codeListWithNumbers, CodeListItemType.Number)).toBe(false);
     });
   });
+
+  describe('updateCodeList', () => {
+    const testRowNumber = 1;
+    const newValue = 'new text';
+
+    it.each(Object.values(CodeListItemTextProperty))(
+      'Returns updated codeList item at the given index',
```

Comment:
Vi bør ha med egenskapen i testbeskrivelsen (ved å referere til den med `%s`). Da kan vi se hvilken egenskap det gjelder hvis én av testene feiler.
```suggestion
      'Returns updated codeList when the %s property is updated',
```